### ISSUE TITLE
st2Auth printing traceback on wrong credentials

### ISSUE THREAD

**Issue by enykeev** [StackStorm/st2#1261]
```
(26140) accepted ('172.168.50.1', 49492)
2015-03-13 05:14:01,784 DEBUG [-] Invalid password for user "test"
2015-03-13 05:14:01,785 AUDIT [-] Invalid credentials provided
2015-03-13 05:14:01,786 ERROR [-] API call failed.
Traceback (most recent call last):
  File "/vagrant/code/stanley/st2common/st2common/models/api/base.py", line 163, in callfunction
    result = f(*args, **kwargs)
  File "/vagrant/code/stanley/st2auth/st2auth/controllers/access.py", line 47, in post
    return self._handle_standalone_auth(request=request, **kwargs)
  File "/vagrant/code/stanley/st2auth/st2auth/controllers/access.py", line 95, in _handle_standalone_auth
    self._abort_request()
  File "/vagrant/code/stanley/st2auth/st2auth/controllers/access.py", line 99, in _abort_request
    pecan.abort(status_code, message)
  File "/vagrant/code/stanley/virtualenv/local/lib/python2.7/site-packages/pecan/core.py", line 119, in abort
    **kw
HTTPUnauthorized: Invalid or missing credentials
172.168.50.1 - - [13/Mar/2015 05:14:01] "POST /tokens HTTP/1.1" 401 461 0.011570
(26140) accepted ('172.168.50.1', 49493)
2015-03-13 05:14:01,801 DEBUG [-] User "wronguser9932" doesn't exist
2015-03-13 05:14:01,801 AUDIT [-] Invalid credentials provided
2015-03-13 05:14:01,802 ERROR [-] API call failed.
Traceback (most recent call last):
  File "/vagrant/code/stanley/st2common/st2common/models/api/base.py", line 163, in callfunction
    result = f(*args, **kwargs)
  File "/vagrant/code/stanley/st2auth/st2auth/controllers/access.py", line 47, in post
    return self._handle_standalone_auth(request=request, **kwargs)
  File "/vagrant/code/stanley/st2auth/st2auth/controllers/access.py", line 95, in _handle_standalone_auth
    self._abort_request()
  File "/vagrant/code/stanley/st2auth/st2auth/controllers/access.py", line 99, in _abort_request
    pecan.abort(status_code, message)
  File "/vagrant/code/stanley/virtualenv/local/lib/python2.7/site-packages/pecan/core.py", line 119, in abort
    **kw
HTTPUnauthorized: Invalid or missing credentials
172.168.50.1 - - [13/Mar/2015 05:14:01] "POST /tokens HTTP/1.1" 401 461 0.007859
```

That's a minor problem, but it attracts too much attention to such an ordinary event.

**Comment by Kami** [comment]
Agreed, we should probably just change the log level to debug.

**Comment by Kami** [comment]
Fixed in #1266
